<launch>

  <!--
  <node name="gripper_controller" pkg="arbotix_controllers" type="gripper_controller">
      <param name="model" value="singlesided"/>
      <param name="invert" value="true"/>
      <param name="pad_width" value="0.0254"/>
  </node>
  -->
  
  <node name="armcontroller" pkg="arm_controller" type="armcontroller" output="screen"/>
 

  </launch>
